gcloud-java-core/src/test/java/com/google/cloud/ServiceOptionsTest.java

```diff
@@ -215,6 +219,16 @@ public void testBaseEquals() {
   }
 
   @Test
+  public void testLibraryName() {
+    assertEquals(LIBRARY_NAME, OPTIONS.libraryName());
+  }
+
+  @Test
+  public void testApplicationName() {
```

Comment:
Library version is not returned when tests are run, I believe due to the fact that classes are loaded directly and not through the jar, so no manifest is there.